examples/daal/cpp_sycl/CMakeLists.txt

```diff
@@ -23,8 +23,13 @@ set(CMAKE_BUILD_TYPE Release)
 set(USE_DPCPP yes)
 set(USE_NEW_IFACES no)
 
+set(custom_options "-fsycl-device-code-split=per_kernel")
```

Comment:
This is out of style, CMake usually uses UPPER_CASE for variables, see how other variables defined

Reply:
Fixed. Please see